### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Andrew-Nielsen**
Added Note to the propose change section to add clarity as some documents require a branch to be created to propose changes.

# Contributor guide pull request

Thanks for contributing to the [Microsoft Learn documentation contributor guide](https://learn.microsoft.com/contribute/?branch=main). Please read these instructions for processing your pull request (PR). Once you've read this text, delete it and write a brief description of the changes you're proposing.

## Quality control

- [ ] 1. **Successful build with no warnings**: Review the build status to make sure **all checks are green** (Succeeded).

- [ ] 2. **#Sign-off**: Once the PR is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the PR. Signing off means the document is ready for review and can be published at any time.

## Merge and publish

- All PRs to this repository are manually reviewed and merged.
- Once all feedback on the PR is addressed, we'll merge the PR into the main branch.

## Need help?

- See our guidance on [Contributing to the contributor guide](/contribute/?branch=main) for help with adding or updating content in this contributor guide.
- Write to the PR reviewer in the PR comments: `@carlyrevier, @jehchow`

### CODE REVIEW

File: Contribute/index.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ Quick edits streamline the process to report and fix small errors and omissions
 4. When you're finished editing, scroll to the bottom of the page. In the **Propose changes** area, enter a title and optionally a description for your changes. The title will be the first line of the commit message. Select **Propose changes** to commit your changes:
```
Comment (jehchow) [inline Contribute/index.md:47]:
```suggestion
4. When you're finished editing, scroll to the bottom of the page. In the **Propose changes** area, enter a title and optionally a description for your changes. The title will be the first line of the commit message. Select **Propose changes** to create a new branch in your fork and commit your changes:
```

File: Contribute/index.md [hunk 2]
```diff
@@ -47,6 +47,8 @@ Quick edits streamline the process to report and fix small errors and omissions
...
 
    :::image type="content" source="media/index/commit-changes.png" alt-text="Propose and commit file changes.":::
+   
+   Note: In other Microsoft documentation, GitHub may require a branch to be created in order to **Propose changes**.
```
Comment (jehchow) [inline Contribute/index.md:51]:
```suggestion

```

Comment (jehchow) [reply Contribute/index.md:51]: @Andrew-Nielsen I think that it'll create a branch every time - mentioning it in the step above could be better than having it as a note.

**Review by jehchow — COMMENTED** [review]
@Andrew-Nielsen Thank you for opening this PR! Agree that we should mention that this process will create a new branch in your fork. Do you agree that it'd be better to just have it as part of the step, as I've suggested in my comments here?

File: Contribute/index.md (hunk 1 above)
Comment (Andrew-Nielsen) [reply Contribute/index.md:47]: Hi @jehchow! Yes, I think that would be fine. Would we need to update the picture if we do that?

Comment (Andrew-Nielsen) [reply Contribute/index.md:47]: Hi @jehchow! Yes, I think that would be fine. Would we need to update the picture if we do that?

File: Contribute/index.md (hunk 2 above)
Comment (Andrew-Nielsen) [reply Contribute/index.md]: @jehchow, after removing the note, it looks like it kept an extra space. Are you seeing the same thing?

**Comment by jehchow** [comment]
@Andrew-Nielsen I've fixed the extra space - if it looks good from your end, please comment `#sign-off` and we'll review and merge!

**Comment by Andrew-Nielsen** [comment]
#sign-off

**Review by jehchow — APPROVED** [review]

**Pull request merged**